ansible/roles/jenkins/tasks/main.yml

```diff
@@ -62,14 +62,14 @@
   when: ansible_os_family == "Debian"
   tags: jenkins
 
-- name: enable the default ssl site
-  command: a2ensite default-ssl 
-  args:
-   creates: /etc/apache2/sites-enabled/default-ssl.conf
-  when: ansible_os_family == "Debian" 
-  notify: reload apache
-  tags: jenkins
-
+# Disabled - SSL certificates need to be configured properly
```

Comment:
I agree and agree in general with the "remove commented crap" prinicple. Will remove.